geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateRegionCommand.java

```diff
@@ -1143,4 +721,26 @@ private boolean isAttributePersistent(RegionAttributes attributes) {
     return attributes != null && attributes.getDataPolicy() != null
         && attributes.getDataPolicy().toString().contains("PERSISTENT");
   }
+
+  private static boolean regionExists(InternalCache cache, String regionPath) {
```

Comment:
Fixed! Thanks for your feedback! Do you mean it would be nice to have a test that would fail if `regionExists` always returns `true`, regardless of whether the region does or doesn't exist? I wrote a very simple test that does this and put it in `CreateRegionCommandTest`.